fuzzing/common.bzl

```diff
@@ -69,7 +69,7 @@ def _fuzzing_corpus_impl(ctx):
     cp_args = ctx.actions.args()
 
     for input_file in ctx.files.srcs:
-        cp_args.add(input_file)
+        cp_args.add(input_file.path)
```

Comment:
Could you also add a test that would fail before this change is introduced? This seems like a tricky case, and we probably want to have a regression test for this :)

Reply:
Even better, a test with a `filegroup` as well would be awesome.

Reply:
Created a new PR #57 for the test, PTAL!